indico/modules/events/registration/__init__.py

```diff
@@ -168,3 +168,15 @@ class RegistrationRole(ManagementRole):
     name = 'registration'
     friendly_name = _('Registration')
     description = _('Grants management access to the registration form.')
+
+
+@template_hook('event-header')
+def _inject_event_header(event, **kwargs):
+    from indico.modules.events.registration.models.forms import RegistrationForm
+
+    event = event.as_event
+    regforms = (event.registration_forms.filter_by(is_deleted=False, is_open=True)
+                     .order_by(db.func.lower(RegistrationForm.title)).all())
```

Comment:
usually we wrap sql queries like this:

```
regforms = (event.registration_forms
            .filter_by(is_deleted=False, is_open=True)
            .order_by(db.func.lower(RegistrationForm.title))
            .all())
```

more readable IMO (one line per chain element)